client/views/hooks/useMembersList.ts

```diff
@@ -36,7 +36,7 @@ export const useMembersList = (
 				options.rid,
 				options.type,
 				{
-					limit: end - start,
+					limit: end,
```

Comment:
This might fix the immediate issue, but the list is supposed to have a `load more...` when you reach 25 users. Applying this fix would be pretty chaotic in bigger channels (like `#general` in open.rocket.chat, with 351039 users).